nilearn/decoding/decoder.py

```diff
@@ -1518,6 +1516,36 @@ def __sklearn_tags__(self):
         tags.estimator_type = "regressor"
         return tags
 
+    def fit(self, X, y, groups=None):
```

Comment:
TODO:
- [ ]  check doc string (as we do not rely on the doc string from _BaseDecoder.fit)